### ISSUE TITLE
Character attacking while dead in tag mode.

### ISSUE THREAD

**Issue by UrsoManso** [Windblade-GR01/Ikemen-GO#349]
https://user-images.githubusercontent.com/77592586/132092199-1a543379-49a2-4bcd-9e3b-cccd39c04605.mp4

Not much to add here honestly, At first I thought that state 18998 was conflicting so I had the idea of changing it but I haven't found it.

[Shishioh.zip](https://github.com/Windblade-GR01/Ikemen-GO/files/7109636/Shishioh.zip)

**Comment by K4thos** [comment]
do a 18998 search in whole chars folder (it's not part of Shishioh, so it must be custom state of one of the other characters present in this match). Uploaded character is not enough to reproduce this issue - at least I couldn't do it on latest pre-0.98 build, with a tag match using 6xShishioh fighting each other.

**Comment by UrsoManso** [comment]
No finds, I also just did a full Shishioh match and the issue was still there.


https://user-images.githubusercontent.com/77592586/132095228-51549ec0-db2d-4f57-95d4-857e6d1a1705.mp4

**Comment by K4thos** [comment]
since I can't reproduce it after several matches I assume it's no lonager a problem on 0.98.

**Comment by K4thos** [comment]
I've managed to reproduce it, will see if I can fix it (unrelated to tag code, P2 somehow can still connect hitdef to dead enemy on some circumstances, will have to investigate why)

**Comment by K4thos** [comment]
nothing to fix here, same happens on mugen 1.1 simul mode:

https://user-images.githubusercontent.com/8928195/132102086-131304bd-f030-45ac-b369-01454d623eaa.mp4

this state is buggy and can execute even post death (I assume due to lack of ctrl trigger):
```
;近距離立ち弱キック
[State -1, Stand Light Kick]
type = ChangeState
value = 235
TriggerAll = (var(59)>3&&Random<=var(59)*100) || fvar(39)
triggerall = RoundSTate = 2 && P2movetype = H
triggerall = !inguarddist
Triggerall = P2statetype != L && P2statetype!=A
triggerall = statetype != A
;trigger1 = stateno = 700 && AnimElemtime(2) >= 1
;trigger1 = floor(EnemyNear(var(58)),GetHitVar(hittime))=5
trigger1 = P2BoDyDist X < 25
```

**Comment by UrsoManso** [comment]
Adding a ctrl trigger apparently fixed the issue, thanks man.
